### ISSUE TITLE
When I try to install the latest 0.21 version of Hugo with Tarball hugo_0.21_macOS-64bit.tar.gz I am getting version 0.18.1

### ISSUE THREAD

**Issue by interglobalmedia** [spf13/hugo#3526]
When I try to install the latest 0.21 version of Hugo with Tarball hugo_0.21_macOS-64bit.tar.gz I amgetting version 0.18.1. I have tried it several times. I have removed hugo and then re-installed. Every time I only get v 0.18.1. I think there is a bug in the Tarball. Thanks!

**Comment by bep** [comment]
I just downloaded and confirmed that that archive is correct:

```
▶ ./hugo env
Hugo Static Site Generator v0.21 darwin/amd64 BuildDate: 2017-05-22T16:11:48+03:00
GOOS="darwin"
GOARCH="amd64"
GOVERSION="go1.8.1"
```

So you must have a conflicting hugo in Path. Try doing `/full/path/hugo` or `./hugo` etc. to make sure you get the "new". Take further questions on the discussion forum.

**Comment by interglobalmedia** [comment]
Hi @bep, Yes, the download was fine after all. I found that I had another install somewhere else from a while ago done with Homebrew. I uninstalled that installation and re-installed with Homebrew again. Now I have the latest v0.21 of Hugo and my site is rendering beautifully thanks to the truncate feature I had implemented but found out through the forum that it didn't work with versions of Hugo < 0.19. My previous version had been v0.18.1. Thanks!
